ControlRoomApplication/ControlRoomApplication/Controllers/RadioTelescopeControllers/RadioTelescopeController.cs

```diff
@@ -791,6 +800,96 @@ private void SensorMonitor()
+                // Check Gate
+                sensors.gate = (SByte)SensorStatusEnum.NORMAL;
+                // if gate open
+                // sensors.gate = (SByte)SensorStatusEnum.ALARM;
+
+                // Check azimuth temp 1
+                sensors.az_motor_temp_1 = (SByte)RadioTelescope.SensorNetworkServer.SensorStatuses.AzimuthTemperature1Status;
+
+                // Check azimuth temp 2
+                sensors.az_motor_temp_2 = (SByte)RadioTelescope.SensorNetworkServer.SensorStatuses.AzimuthTemperature2Status;
+
+                // Check elevation temp 1
+                sensors.el_motor_temp_1 = (SByte)RadioTelescope.SensorNetworkServer.SensorStatuses.ElevationTemperature1Status;
+
+                // Check elevation temp 2
+                sensors.el_motor_temp_2 = (SByte)RadioTelescope.SensorNetworkServer.SensorStatuses.ElevationTemperature2Status;
+
+                // Check weather
+                int windSpeedStatus = RadioTelescope.WeatherStation.CurrentWindSpeedStatus;
+                sensors.weather_station = (SByte)SensorStatusEnum.NORMAL;
+
+                // Tragic wind speed
+                if (windSpeedStatus == 2)
+                {
+                    logger.Info(Utilities.GetTimeStamp() + ": [ControlRoomController] Wind speeds were too high: " + RadioTelescope.WeatherStation.CurrentWindSpeedMPH);
+                    // Might want to consider weather station overrides
+                    sensors.weather_station = (SByte)SensorStatusEnum.ALARM;
+
+                    pushNotification.sendToAllAdmins("WARNING: WEATHER STATION", "Wind speeds are too high: " + RadioTelescope.WeatherStation.CurrentWindSpeedMPH);
+                    EmailNotifications.sendToAllAdmins("WARNING: WEATHER STATION", "Wind speeds are too high: " + RadioTelescope.WeatherStation.CurrentWindSpeedMPH);
+                }
+                // Slightly potentially tragic wind speed
+                else if (windSpeedStatus == 1)
+                {
+                    logger.Info(Utilities.GetTimeStamp() + ": [ControlRoomController] Wind speeds are in Warning Range: " + RadioTelescope.WeatherStation.CurrentWindSpeedMPH);
+                    // Might want to consider weather station overrides
+                    sensors.weather_station = (SByte)SensorStatusEnum.WARNING;
+
+                    pushNotification.sendToAllAdmins("WARNING: WEATHER STATION", "Wind speeds are in Warning Range: " + RadioTelescope.WeatherStation.CurrentWindSpeedMPH);
+                    EmailNotifications.sendToAllAdmins("WARNING: WEATHER STATION", "Wind speeds are in Warning Range: " + RadioTelescope.WeatherStation.CurrentWindSpeedMPH);
+                    
+                }
+                
+                // Check elevation absolute encoder, set to ALERT if timed out
+                sensors.elevation_abs_encoder = (SByte)SensorStatusEnum.NORMAL;
+                if (RadioTelescope.PLCDriver.MotorsCurrentlyMoving())
```

Comment:
Just discovered this bug, if we are only moving the azimuth, this check will pass regardless. We need to find another way to check if the elevation motor is moving or at least should be moving. We could create a new PLCDriver method if necessary.

- [ ] Come up with new logic to check for moving elevation motor

Reply:
Specifying the axis to see if the motor associated with that axis should fix that issue